app/src/main/java/ch/epfl/sweng/swenggolf/offer/ShowOfferActivity.java

```diff
@@ -408,9 +409,21 @@ public void postAnswer(View view) {
                         new Notification(NotificationType.ANSWER_POSTED,
                                 Config.getUser(), offer));
             }
+            sendNotificationToPreviousAnswerers(answers.getAnswerList());
             listAnswerAdapter.notifyDataSetChanged();
         }
+    }
 
+    private void sendNotificationToPreviousAnswerers(List<Answer> answerList) {
```

Comment:
If the creator of the offer leave an answer, he will receive a notification. But he already receives a notification when someone leave an answer. Thus he will receive two notifications when someone leave an answer.

Reply:
Oh yes you're right! I did not think about that.